express-rest-api/server/app.js

```diff
@@ -0,0 +1,27 @@
+var createError = require('http-errors');
+var express = require('express');
+var path = require('path');
```

Comment:
Sorry if I wasnt clear before, could we update all the `var` in all files to `const` pls?

Reply:
Good point! I don't know why express-generator don't use `const`.

Reply:
Yeap maybe I can contribute on that part as well! 😋